gapic/src/main/com/google/gapid/image/ArrayImage.java

```diff
@@ -174,8 +179,10 @@ public Builder flip() {
   public static class RGBA8Image extends ArrayImage {
     private final PixelInfo info;
 
-    public RGBA8Image(int width, int height, int depth, byte[] data) {
-      this(width, height, depth, data, IntPixelInfo.compute(data, true));
+    public RGBA8Image(
+        com.google.gapid.proto.image.Image.ID id, int width, int height, int depth, byte[] data) {
```

Comment:
I actually attempted adding an ID into all these images and decided against it once I started encountering multiple images (your `getCombinedId`). Given that you're then making up identifiers, things are just gross.
What are your thoughts on the standard `equals` & `hashCode` approach in my PR?
It certainly seems like far less code...

Reply:
Done. I think. It's not really less code, but it certainly is cleaner.